docs/pages/reference/backends.mdx

```diff
@@ -172,6 +172,184 @@ teleport:
      etcd_max_client_msg_size_bytes: 15728640
 ```
 
+## PostgreSQL (Preview)
+
+<Details
+  title="Version warning"
+  opened={true}
```

Comment:
@ptgott should we default close this in the v14+ docs and leave it open by default on the v13 docs?